devsetup/edpm/edpm-play.yaml

```diff
@@ -0,0 +1,217 @@
+apiVersion: redhat.com/v1alpha1
```

Comment:
Would it work better to host this file withing the openstack-ansible-operator (or somewhere else) so that the parameters, examples are more close to the codebase? Many of the operators here have the samples they instantiate live in the operators themselves. Like: https://github.com/openstack-k8s-operators/openstack-operator/blob/master/config/samples/core_v1beta1_openstackcontrolplane.yaml

Reply:
Well, openstack-ansibleee-operator is generic. It is not EDPM specific. A sample likely makes sense in openstack-operator, as we make progress on the dataplane CRD/controller. I put it here for the time being as this is purely dev/test setup.

Reply:
I'm good with it.

Reply:
yes